src/transport/MessageHeader.cpp

```diff
@@ -34,6 +34,7 @@
  *  16 bit: | Secure message type                             |
  *  32 bit: | MESSAGE_ID                                      |
  *  32 bit: | Secure Session ID                               |
+ *  16 bit: | Profile ID                                      |
```

Comment:
Should be 32-bits, with 16-bits for the vendor ID and 16-bits for the profile ID, together which form the 32-bit message profile ID.